libr/core/anal.c

```diff
@@ -239,6 +239,17 @@ R_API void r_core_anal_autoname_all_fcns(RCore *core) {
 		}
 	}
 }
+
+static bool blacklisted_word(char* name) {
+	const char * list[] = {"__stack_chk_guard", "__stderrp", "__stdinp", "__stdoutp", 
```

Comment:
uh, wrongly indented the second line, missing spaces after { and before }. ending arrays with NULL is usually desirable, but i guess its fine to iterate on that constantly computed thing.